Intro to open source with GitHub/README.md

```diff
@@ -0,0 +1,22 @@
+# How to work on opensource contribution with Git
```

Comment:
Just make this 'Open Source Contributions with Git'

Reply:
ok